src/scriptdev2/scripts/eastern_kingdoms/zulgurub/boss_jeklik.cpp

```diff
@@ -14,52 +14,52 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-/* ScriptData
-SDName: Boss_Jeklik
-SD%Complete: 85
-SDComment: Some minor improvements are required; Bat rider movement not implemented
-SDCategory: Zul'Gurub
-EndScriptData */
+ /* ScriptData
+ SDName: Boss_Jeklik
+ SD%Complete: 85
+ SDComment: Some minor improvements are required; Bat rider movement not implemented
+ SDCategory: Zul'Gurub
+ EndScriptData */
 
 #include "precompiled.h"
 #include "zulgurub.h"
 
 enum
 {
-    SAY_AGGRO                   = -1309002,
```

Comment:
No idea why'd he'd do so. I can write some fixup for these if we decide the code is good, though.